src/components/link/link.hbs

```diff
@@ -1,9 +1,10 @@
-<!-- 
+<!--
   Copyright IBM Corp. 2016, 2018
 
   This source code is licensed under the Apache-2.0 license found in the
   LICENSE file in the root directory of this source tree.
 -->
 
 <a href="#" class="{{@root.prefix}}--link">Link</a>
-<a href="#" class="{{@root.prefix}}--link" tabindex="-1" aria-disabled="true">Link</a>
+<a class="{{@root.prefix}}--link" tabindex="-1" aria-disabled="true">Link</a>
+<p class="{{@root.prefix}}--disabled">Disabled Link</p>
```

Comment:
Should move the `aria-disabled="true"` here

Reply:
we can completely remove the anchor element without `href` right? or is that a use case we want to support? would we need to add `aria-disabled` on to a text node?

Reply:
I'm just not sure why we are changing it from an `a` element to a `p`. I would think we would keep the same UX pattern that we currently have with our link element, which is adding the aria-label, and adding `pointer-events: none`

Reply:
I think the reason would be to avoid having an `a` without `href`. If we just want the appearance of a disabled link, then it may be better to have a global `.bx--disabled` class rather than styling anchors without hrefs. Just changing the `cursor` and `pointer-events` will not fully disable a link for mobile users if I'm not mistaken, and `touch-action` is unsupported in Safari unfortunately

this article also makes my point and contains some more information about disabling links https://css-tricks.com/how-to-disable-links/

Reply:
I'm fine with that, these are the reasons why we originally didn't support any disabled links, but added it in in the last year or so. I'm good with a global disabled class that can be used on any text element, and it's up to the user to switch to an `a` when it is no longer disabled.